packages/sit-onyx/src/components/OnyxNavItem/future/types.ts

```diff
@@ -0,0 +1,17 @@
+import type { createMenuButton } from "@sit-onyx/headless";
+import type { InjectionKey } from "vue";
+
+export type OnyxNavItemProps = {
+  href?: string;
+  active?: boolean;
+};
+
+/**
+ * [Vue injection key](https://vuejs.org/guide/components/provide-inject) that is provided by the nav bar
+ * to communicate child components whether they should render in mobile or desktop mode.
+ *
+ * @returns `true` if mobile, `false` otherwise
+ */
```

Comment:
Please update the copied comment, to correctly reflect the purpose of this injection key